pipeline/snapcraft/snapcraft_test.go

```diff
@@ -17,24 +18,23 @@ func TestDescription(t *testing.T) {
 }
 
 func TestRunPipeMissingInfo(t *testing.T) {
-	for name, snap := range map[string]config.Snapcraft{
-		"missing summary": {
+	for eerr, snap := range map[error]config.Snapcraft{
```

Comment:
@elopio yeah, I don't like it so much either, but it's ok for small methods IMHO...

thaks